src/hooks/core/chat/useChatUtilsHook.ts

```diff
@@ -238,6 +274,7 @@ function useChatUtilsHook(type: 'SIGNED' | 'ANONYMOUS'): UseChatUtilsHook {
       navigation.navigate('SignedChatScreen');
     }
 
+    console.log('checkpoint 7');
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

It seems like a debug log statement has been left in the code. It's generally a good practice to remove these before committing changes to avoid cluttering the console output in production.
```diff
-    console.log('checkpoint 7');
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->